scripts/install-jstz-cli.sh

```diff
@@ -72,22 +70,30 @@ EOF
 
     echo "Configuring \`jstz\` alias..."
 
-    shell=$(basename "$SHELL")
-    case "$shell" in
-        "bash")
-            shellrc="$HOME/.bashrc"
-            ;;
-        "zsh")
-            shellrc="$HOME/.zshrc"
-            ;;
-        *)
-            cat 1>&2 << EOF
-Unsupported shell: $shell. 
-Please manually add the alias to your shell's configuration file.
-    $shell_alias
-EOF
-            exit 1
-    esac
+    current_shell=$(ps -p $$ -ocomm=)
+
+    echo "SHELL: $SHELL"
+    echo "Detected shell: $current_shell"
+
+    if [[ $SHELL == */bash ]]; then
+        shellrc="$HOME/.bashrc"
+    elif [[ $SHELL == */zsh ]]; then
```

Comment:
Again, clearer to use `$(basename $SHELL)` and store it in a variable for these matches